sdl-core/src/main/scala/io/smartdatalake/workflow/action/DataFrameActionImpl.scala

```diff
@@ -205,8 +205,9 @@ abstract class DataFrameActionImpl extends ActionSubFeedsImpl[DataFrameSubFeed]
             // recreate DataFrame from DataObject if not skipped
             if (!subFeed.isSkipped && (!isRecursive || isDataExisting)) {
               try {
-                logger.info(s"($id) enrichSubFeedDataFrame: getting DataFrame for ${input.id} filtered by partition values ${subFeed.partitionValues.mkString(" ")}" +
-                  subFeed.filter.map(strainer => s" filtered by $strainer").getOrElse(""))
+                logger.info(s"($id) enrichSubFeedDataFrame: getting DataFrame for ${input.id}" +
+                  (if (subFeed.partitionValues.nonEmpty) s" filtered by partition values ${subFeed.partitionValues.mkString(" ")}" else "") +
```

Comment:
this change looks totally unrelated to the Spark Expressions interface. What is the reason behind it?

Reply:
True. It was some small improvement of logging that was on the way.